.travis.yml

```diff
@@ -28,14 +27,14 @@ matrix:
 
 before_install:
   - npm config set spin false
-  - npm install -g bower
+  - npm install -g bower yarn
```

Comment:
Ok that's true. I would try to see if we would be able to install it via repo as it is suggested by yarn team.